.github/workflows/deploy.yaml

```diff
@@ -103,12 +110,31 @@ jobs:
           cli_tests/job.sh
           cli_tests/app.sh
       - name: Login to Docker Hub
-        uses: docker/login-action@v1
+        uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Push to Docker Hub
         uses: docker/build-push-action@v2
         with:
           tags: shipasoftware/ketch:${{github.sha}}
-          push: true
\ No newline at end of file
+          push: true
+
+      - name: Set up QEMU
```

Comment:
These docker emulation actions need to happen before the docker build if I'm not mistaken